### ISSUE TITLE
Launching i3-gnome only launches i3

### ISSUE THREAD

**Issue by naclander** [lvillani/i3-gnome#10]
I have gnome and i3 both installed. Either via `startx`, or via `lightdm` ( `kde-greete` ) gnome and i3 launch separately just fine. 

However, when launching i3-gnome, only i3 seems to launch. I started i3-gnome via `startx` to see if any error messages were printed but I had no success. i3-gnome is installed via the `AUR` package, and I have the same behavior using i3-gnome-git.

I haven't seen a post with a similar problem, so I figured I'd ask here in case anyone else is having this problem. This is on a fresh install of Arch Linux, the gnome and gnome-extra group are installed. The i3 group is also installed. Any advice on how I can further debug this? Also, thanks for the great package, this is definitely better than creating the files myself.

**Comment by 51v4n** [comment]
Hello, sorry for the misconceptions, the deal with `i3-gnome` it's just to get along with GNOME session infrastructure, but what does it mean?

You get the i3 (window manager) as you know *but* still using GNOME desktop envirtonment. The important here is that, for example, if you plug in your headphones, with the traditional use of i3 (only the window manager) you have to load an external o tweak software to get your headphones working, with `i3-gnome` you just plug in and you're just in, because we're under a GNOME session. 

Also if you have a custom GTK+ feel/theme you still have it working under `i3` instead of load a tweak or special software to get a similar behavior, because we're under a GNOME session *without* the graphical components like bars, extensions and so on.
It's more about *session components* than graphical stuff.
Hope this help you.

Cheers!

**TL;DR**
with `i3-gnome` you get along with i3 (window manager) and GNOME (desktop environment) instead of metacity (the *default* window manager of GNOME).

**Comment by 51v4n** [comment]
To debug i suggest you to work directly with the debug session of i3-wm (it can be choosen from login screen)

**Comment by naclander** [comment]
@51v4n Thanks for the reply. I thought i3-gnome would be more simliar to the behavior acheived in this article: http://blog.hugochinchilla.net/2013/03/using-gnome-3-with-i3-window-manager/

My assumption when using i3-gnome was that I wouldn't have to handle display-light, or keyboard-light since those are handled by gnome by default, but according to you that's not the case, correct?

**Comment by 51v4n** [comment]
@naclander Exactly, with i3-gnome we're focus on the window manager under GNOME.
With this in mind you'll feel free to have a more customized environment with the help of GNOME.
Thanks for your interest and enjoy!
